doc/source/user/basics.types.rst

```diff
@@ -96,70 +96,17 @@ The primitive types supported are tied closely to those in C:
 
 
 Since many of these have platform-dependent definitions, a set of fixed-size
-aliases are provided:
+aliases are provided to this site: :doc:`../reference/arrays.scalars`
+under the "Sized aliases" section.
 
-.. list-table::
-    :header-rows: 1
-
-    * - Numpy type
-      - C type
-      - Description
-
-    * - `numpy.int8`
-      - ``int8_t``
-      - Byte (-128 to 127)
-
-    * - `numpy.int16`
-      - ``int16_t``
-      - Integer (-32768 to 32767)
-
-    * - `numpy.int32`
-      - ``int32_t``
-      - Integer (-2147483648 to 2147483647)
-
-    * - `numpy.int64`
-      - ``int64_t``
-      - Integer (-9223372036854775808 to 9223372036854775807)
-
-    * - `numpy.uint8`
-      - ``uint8_t``
-      - Unsigned integer (0 to 255)
-
-    * - `numpy.uint16`
-      - ``uint16_t``
-      - Unsigned integer (0 to 65535)
+In the site above, are listed some types of the table above such as `byte`, 
+`short`, `intc` and `int_`. The corresponding unsigned and signed types
+of these are written too.
 
-    * - `numpy.uint32`
-      - ``uint32_t``
-      - Unsigned integer (0 to 4294967295)
+'float' and 'complex' aliases are listed, as well. The functionality
+for each of these types is descripted in the table above, as well as
+examples with similarities with ``C types``.
```

Comment:
I don't think this text is necessary, as we've already added a link to the section where this is detailed.

Reply:
I read it again and I think that too.